src/Tribe/Query.php

```diff
@@ -242,78 +247,78 @@ public static function pre_get_posts( $query ) {
 
 				if ( ! empty( $query->query_vars['eventDisplay'] ) ) {
 					switch ( $query->query_vars['eventDisplay'] ) {
-						case 'custom':
-							// if the eventDisplay is 'custom', all we're gonna do is make sure the start and end dates are formatted
-							$start_date = $query->get( 'start_date' );
-
-							if ( $start_date ) {
-								$start_date_string = $start_date instanceof DateTime
-									? $start_date->format( Tribe__Date_Utils::DBDATETIMEFORMAT )
-									: $start_date;
+					case 'custom':
```

Comment:
(2/2) here also it looks like this has been reformatted for no particular reason and actually the previous formatting where each `case` block was indented was correct

Reply:
Woops seems like was due the editor let me rollback 💃

Reply:
cool :+1: